docs/book/migrating.md

```diff
@@ -0,0 +1,36 @@
+# Migration Guide
+
+Version 3 is the first major release of `laminas-view` and includes a number of backwards incompatible changes.
+
+## Helper Removals
+
+### Flash Messenger
+
+The flash messenger view helper is no longer present in version 3 and has been migrated to a separate package: [laminas-mvc-plugin-flashmessenger](https://github.com/laminas/laminas-mvc-plugin-flashmessenger). In order to continue to use the flash messenger in your projects, you will need to explicitly require it in your composer dependencies.
```

Comment:
Documentation should be written using the [one sentence per-line method](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line).